src/services/pokemon.ts

```diff
@@ -5,15 +5,26 @@
  * instead of relying on the API to never
  * change.
  */
+
+type PokeApiResults = {
```

Comment:
Basic typing for API results so we can pull data off. This should just match the JSON response of the individual APIs, although the second one is just partial and contains the data that we access. 

The main point of this is to show that the API data matches what we are assigning to `stats`/`Pokemon`.